### ISSUE TITLE
Testing / debugging stage

### ISSUE THREAD

**Issue by farleyknight** [protoman/rockbot#22]
I'm wondering if you have a testing or debugging stage that you use to test things out. This ladder bug will be time consuming to fix if I have to beat the initial stage every time I want to make sure it works. How do you handle issues like this? Would some kind of unit testing make sense, so we can be guaranteed that this kind of functionality doesn't break in the future?

**Comment by protoman** [comment]
Sure. Open game.cpp method quick_load_game(). Set the currentStage to
(SKULLCASTLE5+1) and run the game with the parameter --quickload.
That should avoid the intro and send you directly to that stage. You can
also use the --invencible flag for testing.
Sorry I forgot to tell you.

About the ladder problem, I didn't see that happening before, the one thing
that I knew was you could hold jump button while on the ladder and that
made the player move up/down with the jumping sprite frame :)

## 

Iuri Fiedoruk, Software Developer

**Comment by farleyknight** [comment]
Okay, I will try and come up with a gif that shows the issue I'm having this evening.

**Comment by farleyknight** [comment]
I'm thinking a better alternative to what you describe (about changing quick_load_game) would be to add some more options GAME_FLAGS, so we can specify options like this:

```
./rockbot --quickload --start-with-chaos-city
./rockbot --quickload --start-with-ape-man
./rockbot --quickload --start-with-skullcastle5
```

etc, etc.

**Comment by protoman** [comment]
I made some tests with font to show all characters X and Y. Sadly, due to
low colors and such, the minimal size for font being readable is 8, that
gives you very little space to show information, as it takes too many space.

About the stage, it would be simpler to just pass an integer as the
stage-number, but yeah, need need to add more command-line parameter
options for testing.

## 

Iuri Fiedoruk, Software Developer

**Comment by protoman** [comment]
This will be fixed in #23. Cheats will have an option to allow user to pick any stage, and quickload now shows stage selection.
